### ISSUE TITLE
cellClass broken with release 8.2.0

### ISSUE THREAD

**Issue by gmrodgers** [pivotal-cf/pivotal-ui#511]
This release has broken the functionality of the `cellClass` property of a `TableCell`.  There is no mapping from a `cellClass` to a `className` so no `className` is set.  This issue also causes React to produce warnings that: 

```
Unknown prop `cellClass` on <td> tag. Remove this prop from the element
```

This appears to have been caused by https://github.com/pivotal-cf/pivotal-ui/commit/e3c7d4564c346d545d0f00b29334850234ab9284 (which includes steps to fix this for the flex cell but not regular cells). 

It is also worth noting that when used with custom cells which delegate to a regular cell, similar warnings about `CustomCell` being passed down are produced. It would be nice if all PUI-related properties could be filtered out by the cell when rendering the `td`.

**Comment by charleshansen** [comment]
breaking `cellClass` looks like a real bug, thanks for the catch. I prioritized a story to fix it.

The current philosophy with `CustomCell` is not to filter out any properties because we don't know what people will want to use and you can always filter them out yourself in the `CustomCell` `render` method. Right now, the list of pui props we filter out in the basic cell should be ` ['attribute', 'cellClass', 'displayName', 'index', 'headerProps', 'rowDatum', 'sortable', 'sortBy']` I feel like most of those could be useful in a `CustomCell` if there are using it for multiple columns or multiple tables. Specifically, `attribute`, `index`, `cellClass` and `rowDatum` are useful, maybe even `sortable` or `sortBy`. I'd be up for filtering out `headerProps` or `displayName`, but that doesn't buy you much.

**Comment by gmrodgers** [comment]
That justification sounds good. Also cellClass issue has been fixed in 8.3.0
